data.sql

```diff
@@ -11,3 +11,23 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005-6-12', 1, true, 45);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon','2005-6-7', 7, true, 20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom','1998-10-13', 3, true, 17);
+
+INSERT INTO owners ( full_name, age ) VALUES ('Sam Smith', 34);
+INSERT INTO owners ( full_name, age ) VALUES ('Jenniffer Orwell', 19); 
+INSERT INTO owners ( full_name, age ) VALUES ('Bob', 45);              
+INSERT INTO owners ( full_name, age ) VALUES ('Melody Pond', 77); 
+INSERT INTO owners ( full_name, age ) VALUES ('Dean Winchester', 14); 
+INSERT INTO owners ( full_name, age ) VALUES ('Joddie Whittaker', 38); 
+
+
+INSERT INTO species (name) VALUES('Pokemon');
+INSERT INTO species (name) VALUES('Digimon');
+
+UPDATE animals SET species_id = 1 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 2 WHERE name NOT LIKE '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon'; 
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu'); 
+UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtile', 'Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly move those into `queries.sql` as the project requires so.